src/Intl/Idn/composer.json

```diff
@@ -20,10 +20,8 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
-        "symfony/polyfill-intl-normalizer": "^1.10",
-        "symfony/polyfill-php70": "^1.10",
-        "symfony/polyfill-php72": "^1.10"
+        "php": ">=7.2",
+        "symfony/polyfill-intl-normalizer": "^1.10|^2"
```

Comment:
why `^2` if we don't create a 2.x version ?

Reply:
reverted